frontend/src/static/js/components/webstatus-feature-page.ts

```diff
@@ -307,6 +324,21 @@ export class FeaturePage extends LitElement {
     this.generateFeatureSupportChartOptions();
   }
 
+  async handleUsageBrowserSelection(event: Event) {
```

Comment:
This looks unused.

Reply:
Given we only have one browser, do we need this?

Reply:
No I believe we don't need this - I've went ahead and removed it. 🙂